### PR TITLE
add EU containment

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: internal/store/files/WorldGeosForPlaceRecognition.csv [hunk 1]
```diff
@@ -257,6 +257,7 @@ country/ZAF,Country,South Africa,"Earth,africa",59392255
 country/ZMB,Country,Zambia,"Earth,africa",19473125
 country/ZWE,Country,Zimbabwe,"Earth,africa",15993524
 europe,Continent,Europe,Earth,579000000
+EuropeanUnion,ContinentalUnion,"European Union",Earth,447199800
```
Comment (pradh) [inline internal/store/files/WorldGeosForPlaceRecognition.csv:260]: "European Union,EU"

File: internal/store/files/WorldGeosForPlaceRecognition.csv [hunk 2]
```diff
@@ -17,15 +17,15 @@ country/ASM,Country,American Samoa,"Earth,country/USA,northamerica,oceania",4503
 country/ATF,Country,French Southern Territories,"Earth,africa,antarctica",196
 country/ATG,Country,Antigua and Barbuda,"Earth,northamerica",93219
 country/AUS,Country,Australia,"Earth,oceania",25688079
-country/AUT,Country,Austria,"Earth,europe",8955797
+country/AUT,Country,Austria,"Earth,europe,EuropeanUnion",8955797
```
Comment (pradh) [inline internal/store/files/WorldGeosForPlaceRecognition.csv:20]: For now, can you drop these containment lines?

File: internal/store/files/WorldGeosForPlaceRecognition.csv (hunk 1 above)
Comment (n-h-diaz) [reply internal/store/files/WorldGeosForPlaceRecognition.csv:260]: done

File: internal/store/files/WorldGeosForPlaceRecognition.csv (hunk 2 above)
Comment (n-h-diaz) [reply internal/store/files/WorldGeosForPlaceRecognition.csv]: done

**Review by pradh — APPROVED** [review]

**Pull request merged**